src/lib/index.ts

```diff
@@ -1,3 +1,4 @@
 /* eslint-disable import/prefer-default-export */
 
-export { default as MemoryQueue } from './MemoryQueue';
\ No newline at end of file
+export { default as MemoryQueue } from './MemoryQueue';
+export { default as MongoQueue } from './MongoQueue';
```

Comment:
Please export everything from MongoQueue module. (Reference my comment on `lib/MongoQueue.ts`)

```suggestion
export * from './MongoQueue';
```